src/client/driver/command-executors/client-functions/client-function-executor.js

```diff
@@ -43,12 +46,9 @@ export default class ClientFunctionExecutor {
             });
     }
 
-    // Overridable methods
-    _createReplicator () {
-        return createReplicator([
-            new ClientFunctionNodeTransform(this.command.instantiationCallsiteName),
-            new FunctionTransform()
-        ]);
+    //Overridable methods
+    _addReplicatorTransform () {
```

Comment:
No need to extract methods this way for replicator, just keep it as is. In selector constructor just create `SelectorTransform` and add it to replicator manually.